stylesheets/components/avatar.css

```diff
@@ -0,0 +1,74 @@
+@import url("/stylesheets/main.css");
+
+.avatar {
+  margin: 1rem;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 100%;
+  position: relative;
+}
+
+.avatar-sm {
+  width: 2rem;
+  height: 2rem;
+}
+
+.avatar-lg {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+.avatar-xl {
+  width:4rem;
+  height:4rem;
+}
+
+.avatar-sq {
+  border-radius: 0.2rem;
```

Comment:
try to give the border radius as % in square avatars, since it will always be relative to size of the avatar. for ex- suppose you have an avatar of height and with 0.5rem, then 0.2rem of border radius would make it circular rather than a rounded square

Reply:
I have refactored the code , as suggested.